### ISSUE TITLE
Rglass can easily tolerate the million kPa from a rust but it breaks when the pressure raises a bit more from a plasmafire

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#29740]
(WEB REPORT BY: eneocho REMOTE: 206.221.180.138:7777)
# Revision
7aa4ceb62caa8080093a65e1aa1a4d5b1b0aae99

# Description
Made a rust,and was activating it, around 200000kPa, the traitor telesciences a monkey corpse, the sparks ignite the plasma. The small increase in pressure from the fire breaks my windows when the massive pressure surge from the rust going online (and other rusts in general, which can get easily to a million kPa) couldn't
# Steps to Reproduce
Make RUST.
Enable it.
See how the reinforced glass easily tolerates the massive pressure.
Light the plasma on your rust on fire.
See how the 10kPa increase from the fire breaks your windows and releases the sun-hot plasma.
# What you Expected
For the Rglass that tolerated gigapascals to also tolerate kilopascals.
# What Actually Happened
got cucked outta activating my rust because the tator teleported a fuckin monkey into it and windowcode is fucking garbage

**Comment by DrSnips** [comment]
> Ah yes. Today I will build my rust out of very breakable materials.
> *clueless
Use actual walls and airlocks bucko.

Shitposting aside yeah this is odd.

**Comment by Eneocho** [comment]
I've made rusts out of R-Glass many times and nothing breaks. This is the first time the glass broke, directly because the plasma inside lit on fire (something that normally doesn't happen either because when completely activated the rusts have almost no oxygen inside, or outright don't have oxygen inside). It can tolerate 1 GPa and 800000 Kelvin without cracking, but the extra 2000kPa and 4500 Kelvin from the blasmafire instantly shatters it. I think it's probably a check for windows to crack if pressurized by a plasmafire.

**Comment by Exxion** [comment]
Windows aren't broken by pressure. They're broken by the fire itself.
Always place your windows facing inwards.

**Comment by Eneocho** [comment]
>
>Always place your windows facing inwards.

So I have to build the window on the tiles outside of the area I want to tile? That's fucking ugly as hell. I'd rather risk an eventual plasmaflood every now and then.

**Comment by Exxion** [comment]
Yes, if you want to be safe that's the way to do it. I thought this was common knowledge.
Anyway yeah it would be realistic if fire damaged them from both sides but that's hard to implement efficiently so it doesn't
